### DISCUSSION TITLE
Doesn't detect flatpak tidal-hifi players.

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [altdesktop/playerctl#347] [Q&A]
I have tidal-hifi installed through flatpak, and when I play audio and check with `playerctl --list-all`, it doesn't its player.
How do I make this flatpak app work with playerctl?

**Comment by ferdnyc** [comment]
Looks like the `tidal-hifi` Flatpak build was fixed in March, and should work now. The issue was on the packaging end. (Mastermindzh/tidal-hifi#540)
